src/Security/Acl/Permission/AdminPermissionMap.php

```diff
@@ -111,6 +111,9 @@ class AdminPermissionMap implements PermissionMapInterface
         ],
     ];
 
+    /**
+     * @return array|null interface return type is not correct
```

Comment:
What is the interface return type?

Reply:
```php
/**
     * @return array may return null if permission/object combination is not supported
     */
```

so null is not typehinted

Reply:
can you typehint the interface?

Reply:
no, because it's symfony core

`Symfony\Component\Security\Acl\Permission\PermissionMapInterface`

Reply:
What about opening a PR on symfony ?

Reply:
Would be nice a PR on Symfony, but we can't wait for them to approve, merge an release, better to add it here and maybe add a comment to remove it at some point when symfony has it?

Reply:
Sure, I wasnt saying it was a blocking point.
We can open the symfony PR and add a todo with the link here (Remove this when this PR [link] is merged)

Reply:
https://github.com/symfony/security-acl/pull/58

the PR I did